packages/cli/src/runDeploy.ts

```diff
@@ -41,6 +43,10 @@ export const deployOptions = {
     type: "string",
     desc: "The deployment salt to use. Defaults to a random salt.",
   },
+  awsKmsKeyId: {
+    type: "string",
+    desc: "The AWS KMS keyId. If set, the World is deployed using a KMS signer instead of local private key.",
```

Comment:
tiny thing since they call it "key ID" in the UI
https://docs.aws.amazon.com/kms/latest/developerguide/find-cmk-id-arn.html
```suggestion
    desc: "The AWS KMS key ID. If set, the World is deployed using a KMS signer instead of local private key.",
```